modules/gltf/gltf_document.cpp

```diff
@@ -4606,7 +4606,7 @@ Error GLTFDocument::_parse_lights(Ref<GLTFState> state) {
 			light->outer_cone_angle = spot["outerConeAngle"];
 			ERR_CONTINUE_MSG(light->inner_cone_angle >= light->outer_cone_angle, "The inner angle must be smaller than the outer angle.");
 		} else if (type != "point" && type != "directional") {
-			ERR_CONTINUE_MSG(ERR_PARSE_ERROR, "Light type is unknown.");
+			ERR_PRINT("Light type is unknown.");
```

Comment:
That's changing behavior here. `ERR_CONTINUE_MSG` prints an error, then `continue`.

Reply:
Should we have `ERR_CONTINUE_MSG(true, "Light type is unknown.");` or `ERR_PRINT("Light type is unknown."); continue;`?

Reply:
It's equivalent, I guess we can go with `ERR_CONTINUE_MSG(true, ...)` to make the intention explicit.

Alternatively it would be:
```
} else {
    ERR_CONTINUE_MSG(type != "point" && type != "directional", "Light type is unknown.");
}
```

While at it, could be useful to print `type` in the error for debugging purposes.

Reply:
Reviewed. ERR_CONTINUE_MSG(true, ...) and adding types makes sense.